.github/workflows/build_v2.yml

```diff
@@ -0,0 +1,91 @@
+name: build-v2
```

Comment:
any changes to v1 or v2 workflow would need to be backfilled in the other workflow.
is it possible to have one single workflow file and work with the matrix attribute? https://docs.github.com/en/actions/using-workflows/advanced-workflow-features#using-a-build-matrix

Reply:
implemented! Now we have just one build.yml and one integration_tests.yml that runs for both v1 and v2

Unfortunately this came with the drawback of not being able to run unit tests for v1 on Node12 (as v2 does not support Node12). I think this is fine, thoughts?

If we do want to cover that case, I can write another test that just targets v1 + node12